### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by carolinaniglio**

### CODE REVIEW

**Comment by carolinaniglio** [comment]
:)

File: README.md [hunk 1]
```diff
@@ -137,7 +138,6 @@ To add your own project below, just [edit](https://github.com/wwdc/2019/edit/mas
 |[Oscar Gorog](https://oscargorog.com)|[GitHub](https://github.com/OkiRules/WWDC19-Submission)||SpriteKit, AVFoundation, JSON Parsing|Submitted|
 |Ozan Mirza|[GitHub](https://github.com/ozanmirza1/WWDC-2019-Neural-Networks)|[Youtube](https://youtu.be/1gk3QSbFVpU)|CoreML, Vision, UIKit, QuartzCore|Submitted|
 |[Patcharapon Joksamut](https://twitter.com/Frost_Noxia)|[GitHub](https://github.com/patcharapon-j/AlgoMaze)|[YouTube](https://www.youtube.com/watch?v=m-xPh7gDT9o)|UIKit|Accepted|
-|[Peijun Weng](https://github.com/windstormeye)|[GitHub](https://github.com/windstormeye/WWDC19_brocadeOfLiNationality)||UIKit|Accepted|
```
Comment (MrLotU) [inline README.md]: Why is this line removed?

Comment (carolinaniglio) [reply README.md]: Sorry. It was a mistake. Should I delete this request and make another or will you merge them?

Comment (MrLotU) [reply README.md]: You can add another commit to your branch and add the entry back in, or close this PR and make a new one
